docs/webui.md

```diff
@@ -0,0 +1,206 @@
+---
+title: 'Gravitino webUI'
+slug: /webui
+keyword: webui
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+license: 'Copyright 2023 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.'
+---
+
```

Comment:
Can you please add a simple paragraph to introduce the content of this doc in a general view?